### DISCUSSION TITLE
When will my published repo be available in the npm registry?

### DISCUSSION THREAD

**Discussion by helrayzr** [community/community#46463] [New to GitHub] [answered]
I ran npm publish to Git Packages an hour ago, but when I run npm install in a different project it gives me a 404 Not Found error. I copied the npm install instructions from the Packages sidebar, and I changed visibility to public, so I know I have the right query, it just isn't available.

**Comment by shayan1234554321** [comment, marked as answer]
If you are encountering a "404 Not Found" error when running npm install from a different project, it may mean that your package is not yet available in the npm registry 🤥. 

Typically, it can take a few minutes for your package to be fully processed and available in the npm registry after you run npm publish 🤩. However, in some cases, it may take longer for your package to become available, or it could be that there was an issue with the npm publish process that you ran.

If you are still encountering the "404 Not Found" error after waiting for a reasonable amount of time, you can try the following steps:

1. Check the spelling and casing of the package name in the npm install command.
2. Try running npm install again after a few minutes.
3. Check your npm registry configuration to ensure that it is set up to use the correct registry.

If you are still unable to resolve the issue, you can try reaching out to the npm support team for further assistance 🤗.

**Reply by helrayzr** [reply]
Thank you. it did not update in a reasonable amount of time, so I just shifted to creating an account on npmjs.org and published directly to registry.npmjs.org. It was accessible within minutes.
